src/style.css

```diff
@@ -1 +1,39 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  padding-right: 5px;
+  width: 96px;
+  justify-content: space-between;
+}
+
+.stars__star {
+  display: flex;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+}
+
+.stars--1 :nth-child(1) {
+  background-image: url(images/star-active.svg);
```

Comment:
Use classes instead of complex selectors when possible. Instead of `.stars--1 :nth-child(1)`, consider using a class like `.star--active` on the active stars. This simplifies the CSS and makes it more readable.